tests/templates/test_subroutines/test_adder.py

```diff
@@ -204,6 +204,11 @@ def test_decomposition(self):
         for op1, op2 in zip(adder_decomposition, op_list):
             qml.assert_equal(op1, op2)
 
+    def test_work_wires_added_correctly(self):
+        """Test that no work wires are added if work_wire = None"""
```

Comment:
How the template works with no work wires?

Reply:
When the mod is an exact 2 power, no work qubits are needed.
It is in these cases where None was being included as wire 🐛